WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -126,10 +124,6 @@ - (void)setCurrentTopic:(ReaderAbstractTopic *)topic
     } else {
         [[UserPersistentStoreFactory userDefaultsInstance] setObject:topic.path forKey:ReaderTopicCurrentTopicPathKey];
         [NSUserDefaults resetStandardUserDefaults];
-        
-        dispatch_async(dispatch_get_main_queue(), ^{
-            [[NSNotificationCenter defaultCenter] postNotificationName:ReaderTopicDidChangeNotification object:nil]; 
-        });
```

Comment:
Nitpick. The `[NSUserDefaults resetStandardUserDefaults];` call is duplicated between the two `if` branches. It could be extracted after the `if`.

Reply:
Good catch! I didn't notice that. The method calls has been removed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20137/commits/5bcec77be8980b9901dc2dd69a6cca39094c45b1 since [the documentation](https://developer.apple.com/documentation/foundation/nsuserdefaults/1407708-resetstandarduserdefaults) says this method "shouldn't be used".

Reply:
Ah! Even better!